### PR TITLE
What makes a good report?

### PR DESCRIPTION
**Pull request by rbowen**
Some advice on writing a *good* board report, rather than just checking the box.

### CODE REVIEW

**Review by ShaneCurcuru — COMMENTED** [review]
Great stuff. Should explicitly link to the policy on what to include in reports, so PMC chairs can easily see both the what (policy, which could also be improved to match) and the why (this document):
https://apache.org/foundation/board/reporting#guidelines

File: source/pmc/reporting.md [hunk 1]
```diff
@@ -52,3 +52,42 @@ tool](https://whimsy.apache.org/board/agenda), or committed directly to
...
+sponsors, the press, and the general public to read. Consider that
+audience when writing. For example, if there's a story that you'd like
+the press to tell about your project, you might mention it, and provide
+a quote that you'd like to see in an article.
+
+If there's something you wish to bring to the attention of the board,
+but isn't for public consumption, enclose it in &lt;private&gt; tags so
+that it won't be published in the public minutes.
+
+The report isn't just a once-a-quarter box to check -- It's your
```
[the remark this answers is not in the document]
Comment (rbowen) [reply source/pmc/reporting.md]: Thanks Sebb. Changes made.

**Comment by rbowen** [comment]
@bdelacretaz It is my intent to add some references to "good" reports, and have been reading past minutes to find some good examples.

**Comment by rbowen** [comment]
Thanks, @sebbASF , changes made.

File: source/pmc/reporting.md [hunk 2]
```diff
@@ -52,3 +52,50 @@ tool](https://whimsy.apache.org/board/agenda), or committed directly to
...
+
+The report isn't just a once-a-quarter box to check -- it's your 
+opportunity to tell the story of your project,
+celebrate your accomplishments, ask for help or advice on challenges
+you're facing, and identify places where new contributors might find
+something to work on.
+
+We encourage you to look at [past board
+reports](https://apache.org/foundation/board/calendar.html) for inspiration from
+other projects' reports.
```
Comment (rlenferink) [inline source/pmc/reporting.md:100]: Great addition! Should we maybe link to https://whimsy.apache.org/board/minutes/ if people want to easily select another project and have a look at their past reports ?

**Comment by psteitz** [comment]
This is really nice work.  On first read, I though there were things missing re challenges, but everything is there.  I guess people who know how to write know how to write :)

Re Bertrand's excellent suggestion of including links to good reports and explaining why, I kind of tried to do that for a year or so in my Chairman's reports.  So there may be some examples or at least some inspiration there.  See, e.g. https://svn.apache.org/repos/private/foundation/board/archived_agendas/board_agenda_2018_04_18.txt

That is all old stuff, but the same kind of approach could be taken with newer reports.  Maybe even just using some of the ones I called out might work.

**Pull request merged**